packages/client/src/components/DatasetTable.component.tsx

```diff
@@ -1,19 +1,56 @@
-import { DataGrid, GridColDef } from '@mui/x-data-grid';
+import { DataGrid, GridActionsCellItem, GridColDef, GridRowId } from '@mui/x-data-grid';
 import { useState, useEffect } from 'react';
 import { Dataset, Entry } from '../graphql/graphql';
-import { useEntryForDatasetQuery } from '../graphql/entry';
+import { useDeleteEntryMutation, useEntryForDatasetQuery } from '../graphql/entry/entry';
 import { EntryView } from './EntryView.component';
+import DeleteIcon from '@mui/icons-material/DeleteOutlined';
+import { useConfirmation } from '../context/Confirmation.context';
 
 export interface DatasetTableProps {
   dataset: Dataset;
   additionalColumns?: GridColDef[];
 }
 
 export const DatasetTable: React.FC<DatasetTableProps> = (props) => {
+  const defaultColumns: GridColDef[] = [
+    {
+      field: 'view',
+      headerName: 'View',
+      width: 300,
+      renderCell: (params) => <EntryView entry={params.row as Entry} width={300} />
+    },
+    {
+      field: 'entryID',
+      headerName: 'Entry ID',
+      width: 150,
+      editable: false
+    },
+    {
+      field: 'delete',
```

Comment:
The delete column should not be added directly to the DatasetTable since the DatasetTable is used in other places where the delete option is not supported. This should be passed in via `additionalColumns` from the `DatasetControls` view